pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/parquet/ParquetFileProcessor.java

```diff
@@ -137,12 +137,23 @@ protected List<FileNameWithOffset> getDirectoryListing() throws IOException {
      * @return list of file name and file size in bytes
      */
     static protected List<FileNameWithOffset> getDirectoryListing(String fileSpec, String fileExtension) throws IOException {
-        final Path pathSpec = Paths.get(fileSpec);
+        String[] directories= fileSpec.split(",");
```

Comment:
Please put "," as a constant at the top

Reply:
The file/directory scanning code in both ParqueFileProcessor and RawFileProcessor seems to be identical. 

I suggest either of these - 
1. Create a class FileUtils under utils folder and move the methods there.
2. Create a base class like BaseFileProcessor and move this functionality there

Reply:
Done